sos/report/plugins/ceph_mgr.py

```diff
@@ -61,44 +101,33 @@ def setup(self):
             "version"
         ]
 
-        self.add_forbidden_path([
-            "/etc/ceph/*keyring*",
-            "/var/lib/ceph/*keyring*",
-            "/var/lib/ceph/*/*keyring*",
-            "/var/lib/ceph/*/*/*keyring*",
-            "/var/lib/ceph/osd",
-            "/var/lib/ceph/mon",
-            # Excludes temporary ceph-osd mount location like
-            # /var/lib/ceph/tmp/mnt.XXXX from sos collection.
-            "/var/lib/ceph/tmp/*mnt*",
-            "/etc/ceph/*bindpass*",
-        ])
+        self.add_cmd_output([
+            f"ceph daemon {m} {cmd}" for m in self.get_socks() for cmd in cmds]
```

Comment:
Not knowing `ceph` internals, the `container=..` removal from this command is intentional, right? (as we check the right sockets the container uses)?

Reply:
Correct, containers are used for enablement but these commands run directly on the host.